### ISSUE TITLE
Version 3.12

### ISSUE THREAD

**Issue by zoso1166** [Divested-Mobile/hypatia#91]
Every time I've updated databases on hypatia 3.12, it takes about 10 minutes.  Then to keep it from crashing, I have to go into app settings and clear the storage.  Every single time, I'm running android 12 on LG G8 with 6gb zram and 128 storage, with 29% used

**Comment by SkewedZeppelin** [comment]
maybe try using the mirror option?

here are the current sizes:
```
12M	hypatia-domains-bloom.bin
21M	hypatia-md5-bloom.bin
149M	hypatia-md5-extended-bloom.bin
148K	hypatia-sha1-bloom.bin
5.8M	hypatia-sha256-bloom.bin
```

**Comment by zoso1166** [comment]
I used fdroid for the install, which mirror options?--Sent with mail.com Mail appOn 7/31/24, 2:35 AM SkewedZeppelin ***@***.***> wrote:

   
  maybe try using the mirror option? 
  here are the current sizes: 
  12M	hypatia-domains-bloom.bin
21M	hypatia-md5-bloom.bin
149M	hypatia-md5-extended-bloom.bin
148K	hypatia-sha1-bloom.bin
5.8M	hypatia-sha256-bloom.bin

**Comment by SkewedZeppelin** [comment]
From the top left drop down there is a "database server override" option, try the "Cloudflare mirror" preset.

**Comment by zoso1166** [comment]
Nope, no difference.  Still crashing, went back to 3.09 since 3.12 is still broken --Sent with mail.com Mail appOn 7/31/24, 6:18 PM SkewedZeppelin ***@***.***> wrote:

   
  From the top left drop down there is a "database server override" option, try the "Cloudflare mirror" preset.

**Comment by zoso1166** [comment]
Is this bug going to be fixed?  I'm tired of upgrading to 3.12 then having to roll back after one run to 3.09.  I'm running LG go, Android 12 with 128gb storage and 7gb gram.--Sent with mail.com Mail appOn 7/31/24, 6:34 PM james page ***@***.***> wrote:
Nope, no difference.  Still crashing, went back to 3.09 since 3.12 is still broken --Sent with mail.com Mail appOn 7/31/24, 6:18 PM SkewedZeppelin ***@***.***> wrote:

   
  From the top left drop down there is a "database server override" option, try the "Cloudflare mirror" preset.

**Comment by SkewedZeppelin** [comment]
there was no download or loading code changed between 3.09 and 3.12
